packages/bugsnag_flutter/lib/src/model/event.dart

```diff
@@ -41,22 +83,51 @@ class BugsnagEvent {
         (_unhandled != _originalUnhandled) ? true : null;
   }
 
+  /// Adds a map of multiple metadata key-value pairs to the specified section.
   void addMetadata(String section, MetadataSection metadata) =>
```

Comment:
You amended the type `Map<String, Object>` in `leaveBreadcrumb` - is it correct to use `MetadataSection` here?